<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSelector</class>
 <widget class="QDialog" name="DeviceSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device</string>
  </property>
  <widget class="QListWidget" name="deviceListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
